test/dummy/config/initializers/active_job.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Rails.application.config.active_job.logger = Logger.new(STDOUT)
```

Comment:
why ? this will default to Rails.logger.

Reply:
I don't think that should matter - it's a demonstration of a config option on active_job being ignored if AJ::Base loads too soon.  Rails.application.config.active_job.retry_jitter is another one that's affected.
(Again, that particular option is a bit academic if you're using sidekiq, but my point is that Rails doesn't expect you to load the `Base` part of a framework until the very end of the initialization process)